lib/modules/datasource/rpm/index.ts

```diff
@@ -0,0 +1,220 @@
+import { gunzip } from 'node:zlib';
+import { promisify } from 'util';
+import { XmlDocument } from 'xmldoc';
+import { logger } from '../../../logger';
+import { cache } from '../../../util/cache/package/decorator';
+import { joinUrlParts } from '../../../util/url';
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, Release, ReleaseResult } from '../types';
+const gunzipAsync = promisify(gunzip);
```

Comment:
```suggestion

const gunzipAsync = promisify(gunzip);

```

Reply:
I don't see any difference before and after? Both are `const gunzipAsync = promisify(gunzip);`

Reply:
The whitespace is the relevant change here.

Reply:
updated